float32[] x_cutoff
float32[] y_cutoff
float32[] z_cutoff
---
bool success
